components/automate-ui/src/app/components/page-picker/page-picker.component.html

```diff
@@ -1,35 +1,83 @@
-    (click)="onItemTap(first)"
-    [disabled]="isNull(prev)">
-    <chef-icon>first_page</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-prev"
-    (click)="onItemTap(prev)"
-    [disabled]="isNull(prev)">
-    <chef-icon>chevron_left</chef-icon>
-  </chef-button>
-  <chef-button
-    *ngFor="let item of selectablePages; trackBy: trackBy"
-    class="page-picker-item"
-    [attr.primary]="item === page"
-    [attr.secondary]="item !== page"
-    [attr.disabled]="selectablePages.length <= 1"
-    (click)="onItemTap(item)">
-    {{ item }}
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-next"
-    (click)="onItemTap(next)"
-    [disabled]="isNull(next)">
-    <chef-icon>chevron_right</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-last"
-    (click)="onItemTap(last)"
-    [disabled]="isNull(next)">
-    <chef-icon>last_page</chef-icon>
-  </chef-button>
+  <ng-container *ngIf="!forDesktop">
+    <chef-button secondary
+      class="page-picker-item page-picker-first"
+      (click)="onItemTap(first)"
+      [disabled]="isNull(prev)">
+      <chef-icon>first_page</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-prev"
+      (click)="onItemTap(prev)"
+      [disabled]="isNull(prev)">
+      <chef-icon>chevron_left</chef-icon>
+    </chef-button>
+    <chef-button
+      *ngFor="let item of selectablePages; trackBy: trackBy"
+      class="page-picker-item"
+      [attr.primary]="item === page"
+      [attr.secondary]="item !== page"
+      [attr.disabled]="selectablePages.length <= 1"
+      (click)="onItemTap(item)">
+      {{ item }}
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-next"
+      (click)="onItemTap(next)"
+      [disabled]="isNull(next)">
+      <chef-icon>chevron_right</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-last"
+      (click)="onItemTap(last)"
+      [disabled]="isNull(next)">
+      <chef-icon>last_page</chef-icon>
+    </chef-button>
+  </ng-container>
+
+  <!-- new styles being used only for desktop at the moment -->
+  <ng-container *ngIf="forDesktop">
+    <div class="bx--row">
+      <div class="column information-container ">
+        <div class="per-page-container">
+          <span class="items-per">Items per page: </span>
+            <mat-form-field appearance="none">
+              <mat-select panelClass="chef-dropdown" [(ngModel)]="perPage">
+                <mat-option *ngFor="let amount of maxPageOptions" [value]="amount" 
+                (onSelectionChange)="handleSelectMaxPageItems($event, amount)">{{ amount }}</mat-option>
+              </mat-select>
+            </mat-form-field>
+        </div>
+        <span>{{ itemStartCount }} - {{ itemEndCount | paginationTrueEnd: total }} of {{ total }} items</span>
```

Comment:
paginationTrueEnd is a custom pipe that checks against the end value, preventing it from being larger than the total.